src/github/graphql.ts

```diff
@@ -205,14 +205,30 @@ export interface PullRequestCommentsResponse {
 export interface MentionableUsersResponse {
 	repository: {
 		mentionableUsers: {
-			nodes: [
-				{
-					login: string;
-					avatarUrl: string;
-					name: string;
-					url: string;
-				}
-			];
+			nodes: {
+				login: string;
+				avatarUrl: string;
+				name: string;
+				url: string;
+			}[];
```

Comment:
Now it is properly typed as an array instead of a tuple with one element.
Not directly related to PR, but changed to match `AssignableUsersResponse`